test.hxml

```diff
@@ -0,0 +1,14 @@
+## Neko
+-main TestMain
+-lib munit
+-lib hamcrest
+-lib svg
+-lib openfl
+-lib lime
+
+-cp src
+-cp test
+-neko build/neko_test.n
+-D lime
+-D lime_cairo
+--remap flash:openfl
```

Comment:
I mean adding it at the end, to compile normally and then running the neko (through the -cmd) automatically.